webapi/src/main/scala/org/knora/webapi/routing/Authenticator.scala

```diff
@@ -125,6 +126,49 @@ trait Authenticator {
 
     } yield httpResponse
 
+    def presentLoginFormV2(requestContext: RequestContext)(implicit system: ActorSystem, executionContext: ExecutionContext): Future[HttpResponse] = {
```

Comment:
Does the HTML login form need to be generated by the server, or could it be generated by Knora-ui in the browser?